internal/pgbouncer/postgres.go

```diff
@@ -214,7 +214,7 @@ func generatePassword() (plaintext, verifier string, err error) {
 	// - https://www.pgbouncer.org/config.html#authentication-file-format
 	// - https://github.com/pgbouncer/pgbouncer/issues/508#issuecomment-713339834
 
-	plaintext, err = util.GeneratePassword(32)
+	plaintext, err = util.GenerateASCIIPassword(32)
```

Comment:
:thought_balloon:  Same general thought as mentioned for pgMonitor. I'm thinking this is fine because this password will not be used anywhere else, but I want to make sure that's the case.